DEHPEcosimPro/Views/MainWindow.xaml

```diff
@@ -3,10 +3,12 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:dataTemplate="clr-namespace:DEHPEcosimPro.Views.Rows"
-        xmlns:dxdo="http://schemas.devexpress.com/winfx/2008/xaml/docking" 
+        xmlns:dxdo="http://schemas.devexpress.com/winfx/2008/xaml/docking"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:common="clr-namespace:DEHPCommon.UserInterfaces.Views;assembly=DEHPCommon"
-        xmlns:views="clr-namespace:DEHPEcosimPro.Views" Title="DEHP EcosimPro Adapter" Width="800" Height="450" mc:Ignorable="d">
+        xmlns:objectBrowser="clr-namespace:DEHPCommon.UserInterfaces.Views.ObjectBrowser;assembly=DEHPCommon"
+
+        xmlns:views="clr-namespace:DEHPEcosimPro.Views" Title="DEHP EcosimPro Adapter" Width="1424" Height="747" mc:Ignorable="d">
```

Comment:
Is this the minimum size? If it is, I think it is rather large and a lot of older monitors might not show things correctly.